x-pack/plugins/apm/common/rules/apm_rule_types.ts

```diff
@@ -182,20 +182,24 @@ export function formatAnomalyReason({
   measured,
   windowSize,
   windowUnit,
+  anomalyDetectorTypes,
 }: {
   serviceName: string;
   severityLevel: string;
   measured: number;
   windowSize: number;
   windowUnit: string;
+  anomalyDetectorTypes: AnomalyAlertDetectorType[];
```

Comment:
If we create an alert per detector, this should not take a list of detectors. It should only take one

```suggestion
  detectorType: AnomalyAlertDetectorType;
```

Reply:
Come to think of this again, it might not be a bad idea to batch the alerts together into one. I'm a bit on the fence here tbh

Reply:
As far as I understood, the multiselect is there so we can create a rule for all detectors, if we are going to create one for each, why adding multiselect and just keep what I had in the PR before, when just one option was selected
https://github.com/elastic/kibana/issues/126580

> The new anomaly rule takes a condition of all or a single detector, so the user doesn't have to choose individual rule types for the various anomalies that might occur.
By enabling the user to selected a single detector we keep the existing functionality of being able to select e.g. latency as the detector and determining how to alert on any severity level.
By enabling the user to choose actions on all detector anomalies with the same severity level enables them to have fewer rules that serve as a catch-all

Reply:
We should have a single rule, yes. The question is whether we want that rule to produce a unified alert for all the detectors, or an alert for each detector. I don't know the answer to that question so we may just pick one of them. Afaict we can always change this.